src/actions/gifs.ts

```diff
@@ -120,29 +121,29 @@ export function receiveSearchById(gfyId, gfyItem) {
     };
 }
 
-export function errorSearchById(err, gfyId) {
+export function errorSearchById(err: any, gfyId: string) {
     return {
         type: GifTypes.SEARCH_BY_ID_FAILURE,
         err,
         gfyId,
     };
 }
 
-export function searchScrollPosition(scrollPosition) {
+export function searchScrollPosition(scrollPosition: any) {
```

Comment:
scrollPosition, is a number accordingly to mattermost-webapp source code. (we can replace it here and in the other positions in this same file).

Reply:
👍